packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -1965,3 +1968,38 @@ export const getGridProperties = (layout: Partial<Layout> | undefined, isMultiPl
     layout: reversedGridLayout,
   };
 };
+
+type GetAxisCategoryOrderPropsResult = Pick<ICartesianChartProps, 'xAxisCategoryOrder' | 'yAxisCategoryOrder'>;
+const getAxisCategoryOrderProps = (layout: Partial<Layout> | undefined) => {
+  const result: GetAxisCategoryOrderPropsResult = {};
+
+  const axesById: Record<string, Partial<LayoutAxis> | undefined> = {
+    x: layout?.xaxis,
```

Comment:
>xaxis [](http://example.com/codeflow?start=15&length=5)

after the multiplot change this could by anything from xaxis1 to xaxis20 and so on. Logging a bug. Will fix it with the next set of multiplot changes. 

[https://uifabric.visualstudio.com/iss/_workitems/edit/12908](https://uifabric.visualstudio.com/iss/_workitems/edit/12908)

Reply:
Yes, I have taken the multiplot scenario into account during implementation. In that case, only `axesById` needs to be provided for each plot. The rest of the logic will probably stay the same.